fdbserver/include/fdbserver/StorageServerUtils.h

```diff
@@ -0,0 +1,106 @@
+/*
+ * StorageServerutils.h
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2023 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FDBSERVER_STORAGESERVERUTILS_H
+#define FDBSERVER_STORAGESERVERUTILS_H
+#pragma once
+
+#include "fdbclient/FDBTypes.h"
+#include "flow/flow.h"
+#include "fdbclient/StorageCheckpoint.h"
+
+enum class MoveInPhase {
+	Pending = 0,
+	Fetching = 1,
+	Ingesting = 2,
+	ApplyingUpdates = 3,
+	ReadWritePending = 4,
+	Complete = 5,
+	Cancel = 6,
+	Error = 7,
+};
+
+struct MoveInShardMetaData {
+	constexpr static FileIdentifier file_identifier = 3804366;
+
+	UID id;
```

Comment:
What is `id` for? Is `id` 1:1 mapped to `dataMoveId`?

Reply:
No, it is the identification of the MoveInShard.